### PR TITLE
Logo for Haute Valeur Environnementale

### PR DESCRIPTION
**Pull request by teolemon**
Logo for Haute Valeur Environnementale

### CODE REVIEW

File: html/images/lang/fr/labels/haute-valeur-environnementale-500x500.svg [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="162px" height="162px" viewBox="0 0 162 162" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 63.1 (92452) - https://sketch.com -->
```
Comment (teolemon) [inline html/images/lang/fr/labels/haute-valeur-environnementale-500x500.svg:3]:
```suggestion
```

File: html/images/lang/fr/labels/haute-valeur-environnementale-500x500.svg [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <title>hve_prod_coul</title>
+    <desc>Created with Sketch.</desc>
```
Comment (teolemon) [inline html/images/lang/fr/labels/haute-valeur-environnementale-500x500.svg:5]:
```suggestion
    <desc>Created from official PDF file.</desc>
```

**Comment by stephanegigandet** [comment]
Please rename the suffix to .90x90.svg  (height needs to be 90 in the file name)

**Comment by teolemon** [comment]
Does it have to be 90 for a SVG ?

**Comment by stephanegigandet** [comment]
the SVG itself can stay as is, you just need to change the file name. This is so that we know how wide the image is when displayed at 90px height.

**Comment by teolemon** [comment]
How about the raster ? Does it need a specific name as well or can it be just a confirmation of the ratio ? (90x90 with an actual 500x500)

**Comment by stephanegigandet** [comment]
> How about the raster ? Does it need a specific name as well or can it be just a confirmation of the ratio ? (90x90 with an actual 500x500)

If there is a png file, it should be resized to 90px height, and the name of the file should be .[actual width]x90.png

**Comment by stephanegigandet** [comment]
Only one file is needed for each label. If there is a a SVG, it will be used and the PNG won't be used.

**Pull request merged**